pkg/cmd/generate.go

```diff
@@ -40,10 +40,15 @@ func NewGenerateCmd() *cobra.Command {
 	)
 
 	cmd.Flags().Int32P(
-		FLAG_NTOKENS, "n", 512,
+		FLAG_NTOKENS, "n", 4096,
```

Comment:
The maximum for `code` models went down to just 4,000 sadly: [OpenAI Docs](https://beta.openai.com/docs/models/codex-series-private-beta)